### ISSUE TITLE
npm install failing...?

### ISSUE THREAD

**Issue by tony-kerz** [rethinkdb/rethinkdb-example-nodejs-chat#13]
```
~/g/rethinkdb-example-nodejs-chat (master) $ sw_vers -productVersion 
10.10.3
```
```
~/g/rethinkdb-example-nodejs-chat (master) $ node --version
v0.12.4
```
```
~/g/rethinkdb-example-nodejs-chat (master) $ npm install
-
> bcrypt@0.7.7 install /Users/tony/git/rethinkdb-example-nodejs-chat/node_modules/bcrypt
> node-gyp rebuild

  CXX(target) Release/obj.target/bcrypt_lib/src/blowfish.o
  CXX(target) Release/obj.target/bcrypt_lib/src/bcrypt.o
  CXX(target) Release/obj.target/bcrypt_lib/src/bcrypt_node.o
../src/bcrypt_node.cc:118:18: error: no member named 'Dispose' in 'v8::Persistent<v8::Function, v8::NonCopyablePersistentTraits<v8::Function> >'
        callback.Dispose();
```

**Comment by deontologician** [comment]
Hi @tony-kerz , it looks like this is a v8 version mismatch of some sort. It looks like your node is up to date, but perhaps node-gyp needs to be updated?

**Comment by tony-kerz** [comment]
per [this](https://github.com/TooTallNate/node-gyp/wiki/Updating-npm%27s-bundled-node-gyp) i tried:
```
npm explore npm -g -- npm install node-gyp@latest
```
but still:
```
~/g/rethinkdb-example-nodejs-chat (master) $ npm install
\
> bcrypt@0.7.7 install /Users/tony/git/rethinkdb-example-nodejs-chat/node_modules/bcrypt
> node-gyp rebuild

  CXX(target) Release/obj.target/bcrypt_lib/src/blowfish.o
  CXX(target) Release/obj.target/bcrypt_lib/src/bcrypt.o
  CXX(target) Release/obj.target/bcrypt_lib/src/bcrypt_node.o
../src/bcrypt_node.cc:118:18: error: no member named 'Dispose' in 'v8::Persistent<v8::Function, v8::NonCopyablePersistentTraits<v8::Function> >'
        callback.Dispose();

```

**Comment by deontologician** [comment]
Can you `npm install bcrypt` by itself?

**Comment by deontologician** [comment]
[this](http://stackoverflow.com/a/24275258/180718) suggests deleting your `~/.node-gyp` directory.

**Comment by paulscliu** [comment]
I change the package.json so bcrypt is the latest version then it compiled

**Comment by danielmewes** [comment]
@deontologician should we change the bcrypt version in our package.json or would that have other side effects?

**Comment by deontologician** [comment]
Fixed by a7df95c357bc09f120751ae56090bfe32a4aea35
